packages/discount-code-generator/src/cli.js

```diff
@@ -0,0 +1,217 @@
+import fs from 'fs'
+import npmlog from 'npmlog'
+import PrettyError from 'pretty-error'
+import yargs from 'yargs'
+import json2csv from 'json2csv'
+import csv from 'csv-parser'
+import flatten, { unflatten } from 'flat'
+
+import discountCodeGenerator from './main'
+import { version } from '../package.json'
+
+process.title = 'discountCodeGenerator'
```

Comment:
Sets the process nam. Can be seen on Activity Monitor (Mac) or Task Manager (Windows)